### ISSUE TITLE
gitleaks in megalinter

### ISSUE THREAD

**Issue by raghu1515** [megalinter/megalinter#1527]
I have introduced a dummy leak file and want to scan that gitleaks using megalinter: I have added .gitleaks.toml file at the root of the repository. Have followed the instructions as per megalinter recommendations, but still unable to scan gitleak
Here is my workflow:
name: MegaLinter
'on':
  push:
    branches:
      - raghu-megalinter
jobs:
  build:
    name: MegaLinter
    runs-on: [self-hosted, x64, linux, research]
    steps:
      - name: Checkout Code
        uses: actions/checkout@v2
        with:
          token: ${{ secrets.PAT || secrets.GITHUB_TOKEN }}
          fetch-depth: 0        
      - name: MegaLinter
        uses: megalinter/megalinter@v5
        env:
          APPLY_FIXES: all
          DEFAULT_BRANCH: ${{ inputs.default_branch }}
          VALIDATE_ALL_CODEBASE: true
          ENABLE_LINTERS: REPOSITORY_GITLEAKS
          REPOSITORY_GITLEAKS_CONFIG_FILE: .gitleaks.toml
          #VALIDATE_ALL_CODEBASE: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
      
      # Upload MegaLinter artifacts
      - name: Archive production artifacts
        if: ${{ success() }} || ${{ failure() }}
        uses: actions/upload-artifact@v2
        with:
          name: MegaLinter reports
          path: |
            report
            mega-linter.log 
============================================
.mega-linter.yml file has
# Configuration file for MegaLinter
# # See all available variables at https://megalinter.github.io/configuration/ and in linters documentation
#
APPLY_FIXES: all # When active, APPLY_FIXES must also be defined as environment variable (in github/workflows/mega-linter.yml or other CI tool)
# APPLY_FIXES_EVENT: all, push, pull_request
# APPLY_FIXES_MODE: commit
DEFAULT_BRANCH: ${{ inputs.default_branch }}
# DEFAULT_WORKSPACE: .
VALIDATE_ALL_CODEBASE: true
ENABLE_LINTERS: REPOSITORY_GITLEAKS
REPOSITORY_GITLEAKS_CONFIG_FILE: .gitleaks.toml
# # ENABLE: # If you use ENABLE variable, all other languages/formats/tooling formats will be disabled by default
# # ENABLE_LINTERS: # If you use ENABLE_LINTERS variable, all other linters will be disabled by default
# DISABLE:
#   - CLOUDFORMATION
#     # - COPYPASTE # Uncomment to disable checks of excessive copy-pastes
#       - CSS
#         - EDITORCONFIG
#             - SPELL # Uncomment to disable checks of spelling mistakes
#             - JAVASCRIPT
#               - TEKTON
#               DISABLE_LINTERS:
#                 - JSON_PRETTIER
#                 PRINT_ALL_FILES: false
#                 FILTER_REGEX_EXCLUDE: '(\.automation/test|\.automation/generated|\.github/workflows|docs/javascripts|docs/overrides|docs/json-schemas|flavors|clj-kondo|TEMPLATES)'
#                 JSON_JSONLINT_FILTER_REGEX_EXCLUDE: '(\.vscode/)'
#                 YAML_YAMLLINT_FILTER_REGEX_EXCLUDE: '(templates/\.mega-linter\.yml)'
#                 YAML_PRETTIER_FILTER_REGEX_EXCLUDE: '(templates/\.mega-linter\.yml|mkdocs\.yml)'
#                 YAML_V8R_FILTER_REGEX_EXCLUDE: '(descriptors|templates/\.mega-linter\.yml|\.codecov\.yml)'
#                 BASH_FILTER_REGEX_EXCLUDE: "(lib)"
#                 MARKDOWN_FILTER_REGEX_EXCLUDE: '(license\.md)'
#                 MARKDOWN_MARKDOWN_LINK_CHECK_DISABLE_ERRORS: true
#                SPELL_MISSPELL_FILTER_REGEX_EXCLUDE: '(\.automation/generated|docs/descriptors)'
#                 DOCKERFILE_HADOLINT_ARGUMENTS: "--ignore DL3003 --ignore DL3007 --ignore DL3013 --ignore DL3016 --ignore DL3018 --ignore DL3028 --ignore DL4001 --ignore DL4006 --ignore SC2015 --ignore SC2016 --ignore SC2039 --ignore SC2086 --ignore SC1091"
#                 PYTHON_BANDIT_DISABLE_ERRORS: true
#                 SHOW_ELAPSED_TIME: true
#                 EMAIL_REPORTER_EMAIL: raghavendra.s@philips.com
#                 FILEIO_REPORTER: true
#                 JSON_REPORTER: true
#                 GITHUB_STATUS_REPORTER: false
#                 PLUGINS:
#                   - https://raw.githubusercontent.com/megalinter/megalinter/main/.automation/test/mega-linter-plugin-test/test.megalinter-descriptor.yml
#                  PRE_COMMANDS:
#                    - command: echo "This is MegaLinter PRE_COMMAND on own MegaLinter ! :)"
#                        cwd: "root"
#                         POST_COMMANDS:
#                           - command: echo "This is MegaLinter POST_COMMAND on own MegaLinter ! :)"
#                             cwd: "workspace"
#
===============================================
Please suggest if this correct?

**Comment by nvuillam** [comment]
Gitleaks is not in V5 but will be released in future V6 version, that should happen during the next weeks :)

**Comment by raghu1515** [comment]
Thanks for letting me know that. Iam also having issues with starting megalinter action runner on wsl2, should I need to create new issue for this?

**Comment by Kurt-von-Laven** [comment]
Yes, a new issue would be helpful, @raghu1515. Keeping the issue tracker well organized makes relevant information more accessible to everyone. As the wise @nvuillam once told me, people are more likely to read a thread if the subject immediately conveys relevance to them. Our team runs mega-linter-runner locally on WSL2, so I hope I will be able to help you.

**Comment by Kurt-von-Laven** [comment]
Thanks to the hard work of @nvuillam and others, v6 was just released with gitleaks!
